tests/unit/test_video_rest.py

```diff
@@ -418,7 +416,7 @@ def test_video_publish_edit_publish_again(
             assert res.status_code == 200
             data = json.loads(res.data.decode("utf-8"))
             # Ensure that doi once minted cannot be changed to another value
-            assert data["metadata"]["doi"] == old_doi
+            assert data["metadata"].get("doi") == old_doi
```

Comment:
If there is doi there then this should remain as it was before.

Reply:
But there's no doi, should we remove that part completely?

Reply:
Can you maybe modify the test to create the doi and basically test the case "Publish with DOI, edit, modify DOI, publish again" which should not modify the DOI?